cpp/benchmarks/CMakeLists.txt

```diff
@@ -195,16 +195,16 @@ ConfigureBench(FILL_BENCH filling/repeat.cpp)
 # * groupby benchmark -----------------------------------------------------------------------------
 ConfigureBench(
   GROUPBY_BENCH
-  groupby/group_sum.cu
-  groupby/group_nth.cu
-  groupby/group_shift.cu
+  groupby/group_sum.cpp
+  groupby/group_nth.cpp
+  groupby/group_shift.cpp
   groupby/group_struct_values.cpp
-  groupby/group_no_requests.cu
-  groupby/group_scan.cu
-  groupby/group_rank_benchmark.cu
+  groupby/group_no_requests.cpp
+  groupby/group_scan.cpp
+  groupby/group_rank_benchmark.cpp
```

Comment:
Two requests:
1. Can you also remove the word `benchmark` from this file? It seems to have escaped #10112.
2. Why is this file listed in both the GBench and NVBench sections? I believe it is only an NVBench benchmark.

Reply:
nvbench be like :sob: